README.md

```diff
@@ -1,6 +1,7 @@
 
 ### ![#f03c15](https://placehold.it/15/f03c15/000000?text=+) Community Announcements ![#f03c15](https://placehold.it/15/f03c15/000000?text=+)
-
+* **Apr 26 2019** [We launched a sustainability program](https://typelevel.org/blog/2019/04/24/typelevel-sustainability-program-announcement.html). Please consider supporting us. 
+* **Apr 25 2019** [Cats 2.0.0-M1 is released](https://github.com/typelevel/cats/releases/tag/v2.0.0-M1) with Binary compatibility with 1.x on `cats-core`.
```

Comment:
Weird capitalization.

```suggestion
* **Apr 25 2019** [Cats 2.0.0-M1 is released](https://github.com/typelevel/cats/releases/tag/v2.0.0-M1) with binary compatibility with 1.x on `cats-core`.
```

Maybe also better to say on all but cats-laws?

Reply:
thanks for reviewing. 
Well it also breaks on cats-testkit and alleycats. So 4 of the 6 modules remain BC while the other 3 broke. I struggled finding a concise way to say it.